dmaws/deploy.py

```diff
@@ -65,6 +65,19 @@ def create_version(self, version_label, with_sha=False, description='',
 
         return version_label, bool(created_version)
 
+    def prune_old_versions(self, number_to_keep):
+        assert isinstance(number_to_keep, (int, float))
+
+        versions = self.beanstalk.list_application_versions(self.eb_application)
```

Comment:
Is the order of versions that this returns guaranteed by `beanstalk`? It needs to be ordered with the oldest first for this method to work. (At least to avoid deleting recent versions.)